tiflash: add tls config for tiflash proxy (#4414)
pingcap/docs-cn: enable-tls-between-components.md

        key_path = "/path/to/tiflash-server-key.pem"
        ```

        在 `tiflash-learner.toml` 文件中设置，

        ```toml
        [security]

>         # set the path for certificates. Empty string means disabling secure connectoins.

Reviewer: ```suggestion
        # Sets the path for certificates. An empty string means that secure connections are disabled.
```

- TiFlash（从 v4.0.5 版本开始引入）

    在 `tiflash.toml` 文件中设置：

    ```toml
    [security]
    cert_allowed_cn = ["TiKV-Server", "TiDB-Server"]
    ```

>      在 `tiflash-learner.toml` 文件中设置，

Reviewer: ```suggestion
    在 `tiflash-learner.toml` 文件中设置，
```

>      ```toml

Reviewer: ```suggestion
    ```toml
```
